src/ui/mainwindow.cpp

```diff
@@ -2030,48 +2042,6 @@ void MainWindow::on_actionPrint_Now_triggered()
     QPrinter printer(QPrinter::HighResolution);
     currentEditor()->print(&printer);
 }
-/*
-void MainWindow::on_actionLaunch_in_Chrome_triggered()
-{
-    QUrl fileName = currentEditor()->fileName();
-    if (!fileName.isEmpty()) {
-        QStringList args;
-        args << fileName.toString(QUrl::None);
-        QProcess::startDetached("google-chrome", args);
-    }
-}
-*/
-QStringList MainWindow::currentWordOrSelections()
```

Comment:
Wait, that was actually useful. We lost it along the way.
Can you just fix and keep `currentWordOrSelections()` even if it's not used for now? I'd like to provide it as a Run placeholder later.